include/bl_storage.h

```diff
@@ -238,6 +291,87 @@ NRFX_STATIC_INLINE void read_implementation_id_from_otp(uint8_t *buf)
  * This is a temporary solution until TF-M has access to NSIB functions.
  */
 
+#ifdef CONFIG_NRFX_RRAMC
+NRFX_STATIC_INLINE uint32_t index_from_address(uint32_t address)
+{
+	return ((address - (uint32_t)BL_STORAGE)/sizeof(uint32_t));
+}
+#endif
+
+NRFX_STATIC_INLINE counter_t bl_storage_counter_get(uint32_t address)
+{
+#ifdef CONFIG_NRFX_NVMC
+	return ~nrfx_nvmc_otp_halfword_read(address);
+#elif defined(CONFIG_NRFX_RRAMC)
```

Comment:
```suggestion
#if defined(CONFIG_NRFX_NVMC)
	return ~nrfx_nvmc_otp_halfword_read(address);
#elif defined(CONFIG_NRFX_RRAMC)
```
Use same style, update throughout